src/DonationForms/Blocks/DonationFormBlock/resources/editor/components/DonationFormBlockPreview.tsx

```diff
@@ -1,32 +1,41 @@
 import ModalPreview from './ModalPreview';
-import RevealPreview from './RevealPreview';
 import IframeResizer from 'iframe-resizer-react';
 import {useSelect} from '@wordpress/data';
 
+import '../styles/index.scss';
+
 interface BlockPreviewProps {
     formId: number;
     clientId: string;
-    formFormat: string;
+    formFormat: 'fullForm' | 'modal' | 'newTab' | string;
```

Comment:
I believe that there was also a `reveal` option. What about users that still use it, do we convert that to `modal`?

Reply:
Thank you for all your help with this @alaca!

I made a couple of additional changes to address things I noticed happening, mainly in my updated inspector controls.

1. Both the displayStyle and continueButtonTitle controls were displaying on v2 forms : 51452e25dfa3de97a93d7942db80a9661aee218b + 72a3113affbc0cdfc5d89eb08c7df96b458e2176 + 22f053eabdb636dc73f2de3883620e97e52fc796

2. The edit donation form link was determining the page path based on the template, which did not account for classic v2 forms: a758dad016a8f23dbbf86608fcad7fc810f094fb.

3. The reveal format as you mentioned in this comment : 180267b0d27f5f70c5f31bd33ba765abb058144b.

4. The link on the newTab display needed to be updated. It was previously using the iframe path: e0728016f9f7ff402a61288103d9624d618c9eb4